tests/util/test_dataframe_ops.py

```diff
@@ -52,3 +61,240 @@ def test_parse_check(mock_parse_probtest_csv, setup_csv_files):
 
     pd.testing.assert_frame_equal(df_ref, expected_ref)
     pd.testing.assert_frame_equal(df_cur, expected_cur)
+
+
+def test_force_monotonic():
+    """
+    Test that the function modify the dataframe forcing the values of every line
+    to become non-decreasing monotonic along the columns
+    """
+    # Creation of a DataFrame with MultiIndex on the columns
+    arrays = [
+        ["var1", "var1", "var2", "var2"],
+        ["mean", "max", "mean", "max"],
+    ]
+    columns = pd.MultiIndex.from_arrays(arrays)
+    data = [
+        [1, 5, 2, 7],
+        [3, 2, 1, 9],
+        [2, 8, 5, 4],
+    ]
+    df = pd.DataFrame(data, columns=columns)
+
+    force_monotonic(df)
+
+    # Property verification
+    for stat in df.columns.levels[1]:
+        sub_df = df.loc[:, (slice(None), stat)]
+        assert (sub_df.diff(axis=1).fillna(0) >= 0).all().all()
+
+    # Comparison with expected dataframe
+    expected = pd.DataFrame([[1, 5, 2, 7], [3, 2, 3, 9], [2, 8, 5, 8]], columns=columns)
+    pd.testing.assert_frame_equal(df, expected)
+
+
+def test_compute_rel_diff_basic():
+    """
+    Test that the function is giving the expected values with normal numbers
+    """
+    df1 = pd.DataFrame({"A": [1, 2], "B": [3, 4]})
+    df2 = pd.DataFrame({"A": [1, 1], "B": [3, 5]})
```

Comment:
It is just a minor detail. But I think it would be nice to construct the `df1` and `df2` dataframes in the same way as the result, so something like
```suggestion
    df1 = pd.DataFrame([[1, 3], [2, 4]], columns=["A", "B"])
    df2 = pd.DataFrame([[1, 3], [1, 5]], columns=["A", "B"])
```
or creating the result also from a dictionary.
This also applies for the following tests.